api/v1/client.go

```diff
@@ -1016,7 +1030,7 @@ func (c *storageClient) DebondingDelegations(ctx context.Context, r *http.Reques
 			)
 			return nil, common.ErrStorageError
 		}
-		d.Amount = d.Shares * escrowBalanceDebonding / escrowTotalSharesDebonding
+		d.Amount = uint64(float64(d.Shares) * float64(escrowBalanceDebonding) / float64(escrowTotalSharesDebonding))
```

Comment:
This does not look precise :scream:

Reply:
Would you feel better knowing they are in base units? 😅

Reply:
Not really

Reply:
Alright, let me get that in https://github.com/oasisprotocol/oasis-indexer/issues/113, and try to re-index away some problems now.